LOUPE_Backend/LogService.API/Program.cs

```diff
@@ -1,6 +1,48 @@
+using System.Text.Json.Serialization;
+using LogService.Core.Api.Services;
+using LogService.DataAccessLayer.Context;
+using LogService.DataAccessLayer.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
 var builder = WebApplication.CreateBuilder(args);
+
+var connectionString = builder.Configuration.GetConnectionString("AppDb");
+
+//SERVICES
+builder.Services.AddTransient<ILogService, LogService.Core.Api.Services.LogService>();
```

Comment:
Deze kan je eventueel verplaatsen naar een sub class met een serviceCollectionExtension function, even googlen of vragen hoe.